datafusion/tests/sql.rs

```diff
@@ -188,7 +188,7 @@ async fn parquet_single_nan_schema() {
     let sql = "SELECT mycol FROM single_nan";
     let plan = ctx.create_logical_plan(sql).unwrap();
     let plan = ctx.optimize(&plan).unwrap();
-    let plan = ctx.create_physical_plan(&plan).unwrap();
+    let plan = ctx.create_physical_plan(&plan).await.unwrap();
```

Comment:
we see in the tests that for "regular" DataFusion users, the only API change is the `.await` now required on `create_physical_plan`